legal-api/src/legal_api/models/review.py

```diff
@@ -88,13 +89,19 @@ def get_paginated_reviews(cls, review_filter, mapped_sort_by_column):
             join(Filing, Filing.id == Review.filing_id)
 
         if review_filter.start_date:
-            start_date_utc = LegislationDatetime.as_utc_timezone_from_legislation_date_str(review_filter.start_date)
-            start_of_day = datetime.combine(start_date_utc, datetime.min.time())
-            query = query.filter(Review.submission_date >= start_of_day)
+            start_date = LegislationDatetime.get_start_of_date(review_filter.start_date)
+            query = query.filter(Review.submission_date >= start_date)
```

Comment:
Can you try this? We may not need `get_start_of_date`
```suggestion
            start_date = LegislationDatetime.as_utc_timezone_from_legislation_date_str(review_filter.start_date)
            query = query.filter(Review.submission_date >= start_date.date())
```

Reply:
I'm OK with either, though converting a date-time to a date seems a bit like cheating to me because we assume the time becomes 00:00. Also I have a slight preference for keeping comparisons of like objects (datetime to datetime , not datetime to date), but maybe that doesn't matter here. FWIW.

Reply:
@vysakh-menon-aot unfortunately your suggestion did not work but now the function is consolidated but still needs the start and end of the day